### ISSUE TITLE
Luckys Tale cant start

### ISSUE THREAD

**Issue by Villaman68** [LibreVR/Revive#243]
Using revive I can launch Lucky's Tale but the gamepad (xbox one) wont work. Working in all other games though. Weird thing is Lucky's Tale starts in the background like a ghost overlay,  I have to press the xbox guide button twice to get the game to come to the front?

**Comment by mjgorman** [comment]
Also having issue with Lucky's tale. Tried Steam controller and a PS3 controller with SCP server making it look like xbox360. Neither is detected. Stuck watching Lucky bounce and play on the 'Press A' screen. :-(

**Comment by LibreVR** [comment]
Make sure the Lucky's Tale window is in focus on your monitor or else it won't accept input.

**Comment by mjgorman** [comment]
@LibreVR Thanks for the reply. That was the first thing I tried with no luck. It was working just fine for me the last time I played which was about 2 months ago.

**Comment by Villaman68** [comment]
I got it working 2nd time of loading. It still started as a ghost image but pressing the xbox button once brought it to the front and the controller worked. I checked my monitor and luckys tale was at the front on the desktop as the active window. Not sure if lucky tale was an active window on desktop 1st time round. My gripe now with the game is it loses head tracking really quickly, I'm using revive with rift cat on a galaxy s7. Any ideas on how to solve or improve tracking using this setup. Thanks for the help.
